compass/ocean/tests/overflow/hydro_vs_nonhydro/visualize.py

```diff
@@ -0,0 +1,67 @@
+import matplotlib
+import matplotlib.pyplot as plt
+from netCDF4 import Dataset
+
+from compass.step import Step
+
+matplotlib.use('Agg')
+
+
+class Visualize(Step):
+    """
+    A step for visualizing a cross-section through the fluid
+    descend down the slope.
+
+    """
+    def __init__(self, test_case):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+        """
+        super().__init__(test_case=test_case, name='visualize')
+
+        for grid in ['nonhydro', 'hydro']:
+            self.add_input_file(filename=f'output_{grid}.nc',
+                                target=f'../{grid}/output.nc')
+            self.add_input_file(filename=f'init_{grid}.nc',
+                                target=f'../{grid}/init.nc')
+        self.add_output_file('section_overflow.png')
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        grids = ['hydro', 'nonhydro']
+        nGrids = len(grids)
+        plt.figure(1, figsize=(12.0, 6.0))
+
+        config = self.config
+
+        section = config['horizontal_grid']
+        nx = section.getint('nx')
+        section = config['visualize']
+        time = section.getint('plotTime')
+
+        for j in range(nGrids):
+            grid = grids[j]
+            ncfileIC = Dataset(f'init_{grid}.nc', 'r')
+            ncfile = Dataset(f'output_{grid}.nc', 'r')
+            temp = ncfile.variables['temperature']
+            plt.subplot(2, 1, j + 1)
+            plt.imshow(temp[time, 0:nx, :].T)
+            plt.clim([10, 20])
+            plt.jet()
```

Comment:
```suggestion
            plt.imshow(temp[time, 0:nx, :].T, cmap='cmo.thermal', vmin=10., vmax=20.)
```
This is all you would need to do, in addition to adding `import cmocean`

Reply:
Hi @cbegeman, I changed the `imshow` line and added `import cmocean`, but when I try to commit it tells me
`compass/ocean/tests/overflow/hydro_vs_nonhydro/visualize.py:1:1: F401 'cmocean' imported but unused`
should I just get rid of `import cmocean`?

Reply:
This is how the new plot looks like:
![section_overflow](https://github.com/MPAS-Dev/compass/assets/10819470/42744aee-eb03-46ce-8c5a-f80377699eb2)

Reply:
Done, thank you @xylar! Should I request a re-review?

Reply:
No need.  Thanks!